cli/tests/e2e.test.ts

```diff
@@ -1236,4 +1238,84 @@ describe("e2e tests", function () {
       await verify("0", testTallyFilePath);
     });
   });
+
+  describe("topup message", () => {
+    const user = new Keypair();
+    const tokenAmount = 100;
+    let stateIndex: number | undefined;
+
+    after(async () => {
```

Comment:
No need in async here.
```suggestion
    after(() => {
```